nominees/code-your-future.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Code Your Future",
+  "aliases": [
+    ""
+  ],
+  "description": "A free collaboratively developed open source 8 month web development programme.",
+  "website": "https://codeyourfuture.io/",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://syllabus.codeyourfuture.io/"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceText": "During the course you will learn a technology stack that helps you build anything from a simple website to a powerful database-driven web application.",
```

Comment:
I would suggest having a more specific description that mentions the SDG(s) they are aligned with.

> i.e. SDG 4: Ensure inclusive and quality education for all and promote lifelong learning